.golangci.yml

```diff
@@ -0,0 +1,52 @@
+linters-settings:
+  gofmt:
+    simplify: true
+  misspell:
+    locale: US
+  gofumpt:
+    lang-version: "1.20"
+    extra-rules: true
+  forbidigo:
+    forbid:
+      - context\.WithCancel$
+      - ^print.*$
+      - panic
+  errorlint:
+    errorf-multi: true
+
+linters:
+  disable-all: true
+  enable:
+    - bidichk
+    - errcheck
+    - gofmt
+    - goimports
+    - gosimple
+    - govet
+    - ineffassign
+    - misspell
+    - revive
+    - staticcheck
+    - typecheck
+    - unused
+    - whitespace
+    - gofumpt
+    - errorlint
+    - forbidigo
+
+run:
+  timeout: 5m
+  go: "1.20"
+
+issues:
+  exclude-rules:
+    # gin force us to use string as context key
+    - path: server/store/context.go
+      linters:
+        - staticcheck
+        - revive
+
+    # let cli use print and panic
+    - path: "cmd/*|cli/*"
+      linters:
+        - forbidigo
```

Comment:
Could you add the ci job as well please

Reply:
Sure :+1:

Reply:
https://github.com/woodpecker-ci/autoscaler/pull/6